docs/setup.md

```diff
@@ -82,7 +82,8 @@
 3. **Start Gulp to watch JS and CSS**
 
       $ `gulp`
-
+      - if you are getting error message "Error: EACCES: permission denied, open '/home/rlot/.babel.json'", use:
+      `BABEL_DISABLE_CACHE=1 gulp`
```

Comment:
Let's put this in docs/troubleshooting.md (and add a link to that file from the end up setup.md)